### DISCUSSION TITLE
partVolume math

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/Taerobee#76] [General]
# X-1

|   |          | ***diameter (m)*** | ***height (m)*** | ***math*** | ***radius (m)*** | ***m²*** | ***Liters*** |
| :--------- | :--------: | ------------------: | ----------------: | :----------: | ----------------: | --------: | ------------: |
|  X1-body   | cylinder | 1.77               | 2.39             | πr²h       | 0.885            | 5.88077  | 5880.78      |
| X1-crew   | cylinder | 1.25               | 2.64             | πr²h       | 0.625            | 3.23977  | 3239.77      |
| X1-engine | cylinder | 0.625              | 0.74             | πr²h       | 0.3125           | 0.22703  | 227.03       |
| X1-tail   | cone     | 1.25               | 3.28             | (πr²h)/3   | 0.625            | 1.34172  | 1341.73      |

**Comment by zer0Kerbal** [comment]
# Bumper

|                      |           | ***diameter (m)*** | ***height (m)*** | ***width (m)*** | ***math*** | ***radius (m)*** | ***m²*** | ***Liters*** |
| :-------------------- | :---------: | ------------------: | ----------------: | ---------------:  | :----------: | ----------------: | --------: | ------------: |
| bumper-body          | cylinder  | 0.625              | 1.09             |                 | πr²h       | 0.3125           | 0.33441  | 334.41       |
| bumper-control       | cylinder  | 0.625              | 0.09             |                 | πr²h       | 0.3125           | 0.02761  | 27.62        |
| bumper-engine        | cylinder  | 0.625              | 1.34             |                 | πr²h       | 0.3125           | 0.41111  | 411.11       |
| bumper-engine-unclad | cylinder  | 0.625              | 1.33             |                 | πr²h       | 0.3125           | 0.40804  | 408.04       |
| bumper-fin           | rectangle | 0.46               | 1.42             | 0.02            | L*H*W      | 0.23             | 0.01306  | 13.07        |
| bumper-nose          | cone      | 0.625              | 2.36             |                 | (πr²h)/3   | 0.3125           | 0.24135  | 241.35       |

**Comment by zer0Kerbal** [comment]
| ***Taerobee***           |            | ***diameter (m)*** | ***height (m)*** | ***width (m)*** |     ***math***     | ***radius (m)*** | ***m²*** | ***Liters*** |
| :----------------------- | :--------: | -----------------: | ---------------: | --------------: | :----------------: | ---------------: | -------: | -----------: |
| taerobee-adapter-short   | trunc.cone |              0.625 |              0.1 |         0.15625 | (1/3)πh (r²+rR+R²) |           0.3125 |  0.01790 |         17.9 |
| taerobee-adapter-tall    | trunc.cone |              0.625 |              0.3 |         0.15625 | (1/3)πh (r²+rR+R²) |           0.3125 |  0.05369 |        53.69 |
| taerobee-aerobee         |  cylinder  |              0.625 |             0.43 |                 |        πr²h        |           0.3125 |  0.13192 |       131.93 |
| taerobee-control         |  cylinder  |              0.625 |              0.1 |                 |        πr²h        |           0.3125 |  0.03068 |        30.68 |
| taerobee-decoupler       |  cylinder  |              0.625 |             0.19 |                 |        πr²h        |           0.3125 |  0.05829 |         58.3 |
| taerobee-despin          |  cylinder  |              0.625 |              0.1 |                 |        πr²h        |           0.3125 |  0.03068 |        30.68 |
| taerobee-fin-large       | rectangle  |               0.36 |             0.72 |            0.02 |       L*H*W        |           0.1800 |  0.00518 |         5.19 |
| taerobee-fin-small       | rectangle  |               0.37 |             0.16 |            0.04 |       L*H*W        |           0.1850 |  0.00237 |         2.37 |
| taerobee-nosecone        |    cone    |              0.625 |             0.98 |                 |      (πr²h)/3      |           0.3125 |  0.10022 |       100.23 |
| taerobee-parachute       |    cone    |              0.625 |              0.3 |                 |      (πr²h)/3      |           0.3125 |  0.03068 |        30.68 |
| taerobee-science         |  cylinder  |              0.625 |              0.3 |                 |        πr²h        |           0.3125 |  0.09204 |        92.04 |
| taerobee-sustainertank-3 |  cylinder  |              0.625 |             2.88 |                 |        πr²h        |           0.3125 |  0.88357 |       883.58 |
| taerobee-tinytim         |  cylinder  |              0.625 |             1.14 |                 |        πr²h        |

**Comment by zer0Kerbal** [comment]
what do you think @JamesErvin-5 and @staticalliam7?

**Reply by ghost** [reply]
I'm not too experienced with part volumes...

**Comment by vghfr** [comment]
I'll take a look in a little while, I've gotta finish up some homework

**Comment by zer0Kerbal** [comment]
# the biggest outliers

X1-body, X1-tail 
---
along with taerobee-nosecone, 

## cause of release delay

there was much more work than I had suspected, including attempting to add two new parts.

**Comment by zer0Kerbal** [comment]
found the math for the adapters - volume of a truncated cone.

**Comment by zer0Kerbal** [comment]
![image](https://user-images.githubusercontent.com/39887717/152481903-938732fb-4cc9-4750-9ff4-e7e122d4c258.png)
